google/cloud/forseti/common/util/file_uploader.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2018 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Utility functions for file uploads."""
+
+
+import tempfile
+
+from google.cloud.forseti.common.data_access import csv_writer
+from google.cloud.forseti.common.gcp_api.storage import StorageClient
+from google.cloud.forseti.common.util import logger
+from google.cloud.forseti.common.util import parser
+
+
+LOGGER = logger.get_logger(__name__)
+
+
+def upload_json(data, gcs_upload_path):
+    """Upload inventory summary/violation data in json format.
+
+    Args:
+        data (dict): the data to upload
+        gcs_upload_path (string): the GCS upload path.
+    """
+    with tempfile.NamedTemporaryFile() as tmp_data:
+        tmp_data.write(parser.json_stringify(data))
+        tmp_data.flush()
+        storage_client = StorageClient()
+        storage_client.put_text_file(tmp_data.name, gcs_upload_path)
+
+
+def upload_csv(data, gcs_upload_path):
+    """Upload inventory summary/violation data in csv format.
+
+    Args:
+        data (dict): the data to upload
+        gcs_upload_path (string): the GCS upload path.
+    """
+    with csv_writer.write_csv(
+        resource_name='inv_summary', data=data, write_header=True) as csv_file:
```

Comment:
If this is to be general util, then `resource_name` should not be `inv_summary`.

Reply:
You are correct. Fixed.